css/style.css

```diff
@@ -461,19 +460,49 @@ footer {
 /* media querry for small screens */
 @media only screen and (max-width: 768px) {
   /* nav and icon display */
-  .toggle-menu nav {
-    display: flex;
-    justify-content: space-between;
-    align-items: baseline;
-  }
-
   .welcome {
     display: block;
   }
 
-  .small {
-    display: inline-block;
-    padding-right: 30px;
+  .mobile-menu {
+    position: absolute;
+    top: 0;
+    right: -100%;
+    width: 100%;
+    height: 100vh;
+    background-color: rgb(255, 255, 255);
+    transition: right 1s ease-in;
+  }
+
+  .show {
+    right: 0;
+  }
+
+  .mobile-menu i {
+    position: absolute;
+    top: 1.5em;
+    right: 2em;
+    color: #939f94;
+    font-size: 28px;
```

Comment:
- [ ] While you did make substantial efforts in making the mobile view, there are a few things I noticed that do not give in to proper web accessibility which will give a bad user experience. I'll try as much as possible to highlight them below with relevant screenshots. Kindly use your chrome developer tools to access this yourself after the changes, to be optimistic that it meets the desired design. your hard work is noticed 👏 

- At first glance, the mobile view setting is not responsive relative to the screen size. from the view below, you'd notice a margin on the right and a displaced mobile `nav`. this turns to normal after the hamburger is clicked, which should not be so, but otherwise when the `X` button is clicked.  This [Resource](https://dev.to/ljcdev/easy-hamburger-menu-with-js-2do0) will be helpful 

# View on `375` initial :
![image](https://user-images.githubusercontent.com/50990925/215661609-12c7e2bf-6ebd-4783-97e1-0d96fe35eefe.png)

# View when the hamburger is clicked(Should be the `X` button)
You can see the margin by the right. Kindly fix this to have the same design as your earlier approved mobile design. you'd agree with m that this does not give a good user experience. well done so far 👏 
![image](https://user-images.githubusercontent.com/50990925/215661830-4d68335e-8ab1-4385-a9e2-3e79aeb51b1f.png)